app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
-class ApplicationController < ActionController::Base
-end
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+

```

Comment:
- [ ] Despite being logged out I am still able to visit different pages. Kindly fix this. See example below.

  logged out but can visit foods list page | logged out but can visit recipe creation page
  :---:|:---:
  ![iro](https://user-images.githubusercontent.com/54848363/202840794-b4c07547-cb05-4109-b09c-17e4a1f8c01f.jpg) | ![rurie](https://user-images.githubusercontent.com/54848363/202840795-b0d71346-3147-4ffa-b206-0a523b4cc812.jpg)

  You can fix this by adding the following helper to this `application_controller.rb` file. This way only signed-in users can visit certain parts of the app.
  ```
  before_action :authenticate_user!
  ```